opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -259,14 +265,22 @@ message Span {
     // then no attributes were dropped.
     uint32 dropped_attributes_count = 5;
 
-    // Flags, a bit field. 8 least significant bits are the trace
-    // flags as defined in W3C Trace Context specification. Readers
-    // MUST not assume that 24 most significant bits will be zero.
-    // When creating new spans, the most-significant 24-bits MUST be
-    // zero.  To read the 8-bit W3C trace flag (use flags &
-    // SPAN_FLAGS_TRACE_FLAGS_MASK).  [Optional].
+    // Flags, a bit field.
+    //
+    // Bits 0-7 (8 least significant bits) are the trace flags as defined in W3C Trace
+    // Context specification. To read the 8-bit W3C trace flag, use
+    // `flags & SPAN_FLAGS_TRACE_FLAGS_MASK`.
     //
     // See https://www.w3.org/TR/trace-context-2/#trace-flags for the flag definitions.
+    //
+    // Bits 8 and 9 represent the 3 states of whether the link is remote. The states
```

Comment:
Do we want to document which bit represents what (not by adding any enums, but just document)?

Reply:
The bits are documented on line 346 and 347.

Reply:
@bogdandrutu can you please let us know if the documentation on lines 346 and 347 are sufficient? Thanks

Reply:
Ping @bogdandrutu

Reply:
This is done, only one comment left to the flag.